client/templates/reporting/standard-reports/bespoke-report-body/unpaid-attendance-detailed.njk

```diff
@@ -0,0 +1,56 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% set poolTrials = tables[0].rows %}
+
+{{ govukTable({
+  firstCellIsHeader: true,
+  head: tables[0].headers
+}) }}
+
+{% for poolTrial in poolTrials %}
+<h3 class="govuk-heading-m"><a href="{{ poolTrial.poolTrialLink }}">{{ poolTrial.header }}</a></h3>
+{% for item in poolTrial.dates%}
+<table class="govuk-table govuk-!-margin-bottom-0 govuk-!-margin-top-5">
+  <tbody class="govuk-table__body">
+    <tr class="govuk-table__row">
+        <td class="govuk-table__cell" colspan="6"><p class="govuk-body-l govuk-!-margin-0">{{ item.date | dateFilter("YYYY,MM,DD", "dddd D MMMM YYYY") }}</p></td>
+    </tr>
+    {% for row in item.values %}
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell mod-!-width-one-sixth"><a href="{{ url('juror-record.overview.get', { jurorNumber: row.jurorNumber }) }}" class="govuk-link">{{row.jurorNumber}} </a></td>
+      <td class="govuk-table__cell mod-!-width-one-sixth">{{ row.firstName }}</td>
+      <td class="govuk-table__cell mod-!-width-one-sixth">{{ row.lastName }}</td>
+      <td class="govuk-table__cell mod-!-width-one-sixth"><a href="{{ url('reports.financial-audit.get', { auditNumber: row.auditNumber }) }}" target="_blank">
+              {{ row.auditNumber }}
+            </a></td>
```

Comment:
So for this link, if the audit number begins with 'P' it should link to the 'pool-attendances-audit' report and if it is 'J' then should link to 'jury-attendance-audit' report

Reply:
Done